src/views/confirmation/confirmation.styles.ts

```diff
@@ -0,0 +1,82 @@
+import { createUseStyles } from "react-jss";
+
+import { Theme } from "src/styles/theme";
+
+const useConfirmationStyles = createUseStyles((theme: Theme) => ({
+  card: {
+    padding: theme.spacing(2),
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    gap: theme.spacing(1),
+    [theme.breakpoints.upSm]: {
+      margin: [theme.spacing(3), 0],
+      padding: theme.spacing(3),
+    },
+  },
+  icon: {
+    marginTop: theme.spacing(1),
+    [theme.breakpoints.upSm]: {
+      marginBottom: theme.spacing(2),
+    },
+  },
+  chainsRow: {
+    width: "100%",
+    display: "flex",
+    alignItems: "center",
+    flexDirection: "column",
+    justifyContent: "space-between",
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(2),
+    [theme.breakpoints.upSm]: {
+      flexDirection: "row",
+      marginTop: theme.spacing(2),
+      marginBottom: theme.spacing(4),
+    },
+  },
+  chainBox: {
+    display: "flex",
+    gap: theme.spacing(1),
+    width: 239,
+    backgroundColor: theme.palette.grey.light,
+    justifyContent: "center",
+    padding: theme.spacing(1.25),
+    borderRadius: theme.spacing(7),
```

Comment:
`theme.spacing` should only be used for spacers (margin, padding, gap).

Reply:
I'll change every `borderRadius` in a new pr